README.md

```diff
@@ -58,6 +58,7 @@ _Access policies have been deprecated. Please [read this](./access/README.md) fo
 - [Check blast radius](./plan/check-blast-radius.rego)
 - [Check sanitized value](./plan/check-sanitized-value.rego)
 - [Deny on proposed runs but warn on tracked runs](./plan/deny-proposed-runs-warn-track-runs.rego)
+- [Do not destroy S3](./plan/do-not-destroy-s3.rego)
```

Comment:
AWS S3 is a service with many different resource types. let's be specific.

```suggestion
- [Do not destroy S3 bucket](./plan/do-not-destroy-s3-bucket.rego)
```

Reply:
will resolve this as I will change it to stateful resources